### ISSUE TITLE
What about IPFS integration?

### ISSUE THREAD

**Issue by D4708** [FreeTubeApp/FreeTube#1164]
As FreeTube application written using node-js, I believe it's not hard to integrate IPFS server to make the content more distributed. Also it would be nice to have the sharing feature that links to FreeTube local app instead of sharing YouTube links. And finally, distributed file storage would be helpful for invidious.snopyta.org project as more cost-effective model.

I wish to see the next FreeTube release with IPFS integration!
https://github.com/ipfs/js-ipfs

**Comment by qbitroot** [comment]
But FreeTube is "An Open Source **YouTube** app for privacy", not a generic Internet media player. I think this feature is superfluous.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I agree that this is not something we should work on. 

@PikachuEXE @ChunkyProgrammer thoughts?

**Comment by PikachuEXE** [comment]
> FreeTube is an open source desktop YouTube player built with privacy in mind.

I don't think we should consider anything unrelated until we got 1.0/2.0 released and stablized.
